pytorch_lightning/accelerators/accelerator.py

```diff
@@ -379,7 +379,7 @@ def optimizer_state(self, optimizer: Optimizer) -> Dict[str, torch.Tensor]:
         return getattr(self.training_type_plugin, 'optimizer_state', lambda x: x.state_dict())(optimizer)
 
     def on_save(self, checkpoint: Dict[str, Union[Any, torch.Tensor]]) -> Dict[str, Union[Any, torch.Tensor]]:
```

Comment:
@SeanNaren Would need to update the return type annotation as well.

Reply:
Where exactly?